### ISSUE TITLE
[Request] Implement a check for correct file downloading

### ISSUE THREAD

**Issue by sideloading** [w0d4/yt-backup#31]
**Describe the bug**
Some videos are marked as downloaded in the database (`downloaded = 1`), when they are missing 
`size`, `resolution` and `runtime` (all NULL). These videos aren't saved to the archive.log, but since they are marked downloaded in the DB it does not re-download. 

**To Reproduce**
I've left the script running for a couple of days, and notice it a few times within the day. Unfortunately I'm not sure how to reproduce.

**Expected behavior**
Download correctly, if download/file checks fail don't mark as downloaded.

**Debug log**
If applicable, add debug log of your problem with --debug
I will run the service with the debug command today and see if I come across it again.

Here is the non-debug log:
```
2020-05-29 01:55:29,110 - yt-backup - INFO - Added new video videoID to DB.
2020-05-29 02:07:56,775 - yt-backup - INFO - Video videoID - TITLE is not yet downloaded. Downloading now.
2020-05-29 02:08:55,008 - yt-backup - ERROR - Could not find video size for video /mnt/shares/yt-backup/tmp/channelname [id]/20200528 video title.f136.mp4
2020-05-29 02:08:55,045 - yt-backup - ERROR - Could not find resolution for video /mnt/shares/yt-backup/tmp/channelname [id]/20200528 video title.f136.mp4
2020-05-29 02:08:55,046 - yt-backup - ERROR - Could not find size for video videoID 
2020-05-29 02:08:55,063 - yt-backup - INFO - Video videoID is downloaded.
```

**Additional context**
These are my additional-options. Note I am using aria2c as an external downloader to speed up downloads for supported streams. 
```
"additional-options": "--write-info-json --add-metadata --write-thumbnail --prefer-ffmpeg --merge-output-format mkv --no-overwrites --external-downloader aria2c --external-downloader-args '-j 8 -s 8 -x 8'"
```

**Comment by w0d4** [comment]
Will implement this in a later version.

**Comment by w0d4** [comment]
Implemented in b11bcd9
